lessons/projects/asteroids/index.md

```diff
@@ -35,11 +35,11 @@ První krok bude naprogramovat vesmírnou loď, která půjde ovládat klávesni
 
 * Vesmírnou loď bude reprezentovat objekt třídy `Spaceship`.
 * Každá loď má vlastní atributy `x` a `y` (pozice),
-  `x_speed` a `y_speed` (rychlost), `rotation` (úhel natočení),
-  `sprite` (obrázek pro Pyglet) a `window` (okno, ve kterém se hraje).
+  `x_speed` a `y_speed` (rychlost), `rotation` (úhel natočení) a
+  `sprite` (obrázek pro Pyglet).
```

Comment:
Holky se často ptají, jaký je rozdíl mezi spritem a obrázkem - proč nevykreslujeme přímo obrázek, ale děláme vůbec nějakou instanci Spritu. Co sem napsat něco jako tohle?
```
`sprite` (2D objekt v Pygletu s polohou, rotací a obrázkem)
```

Reply:
To mi dává smysl. Díky